windrose/windrose.py

```diff
@@ -642,6 +642,8 @@ def bar(self, direction, var, **kwargs):
                     zorder=zorder,
                     **kwargs,
                 )
+                # needed so the the line of the rectangle becomes curved
+                patch.get_path()._interpolation_steps = 100
```

Comment:
I 100% agree on that being not optimal and originally private, but since it's even in the docs I think that's the only way to get there?

I still do think a straight line is technically incorrect in a polar projection.

Reply:
> I still do think a straight line is technically incorrect in a polar projection.

For our eyes?  Yes. I agree. But in theory the "data" is correct otherwise the higher interpolation steps would still produce a straight line, right?

Anyway, I did not mean that this was the problem as for a graph what we see is the important part. I think that mpl is probably choosing non-ideals defaults for us in this case.